packages/telemetry/test/session-event.test.js

```diff
@@ -411,6 +411,29 @@ describe('Session event', () => {
 		});
 	});
 
+	describe('config.integrations + optionalIntegrations + totalNumOfIntegrations', () => {
```

Comment:
Test coverage, thank you!!  🎉 🥳 

Would you mind adding a case where all the integrations are falsy?  Feels like one of those edge cases that I could break without realizing it 😅